openregister-conformance

```diff
@@ -10,7 +10,9 @@ import pytest
 
 parser = argparse.ArgumentParser()
 parser.add_argument("endpoint", nargs="+", help="URL(s) of register to test")
```

Comment:
do we want to (either in this PR or in another) remove the `nargs="+"` bit that allows targetting multiple endpoints from one call? We don't really use this feature anyway.